src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -16,7 +16,7 @@
     <CodeCoverageEnabled Condition="'$(SkipTests)' != 'true' and '$(RunningOnUnix)' != 'true' and '$(Coverage)' == 'true' and '$(Performance)' != 'true'">true</CodeCoverageEnabled>
     <CoverageReportDir Condition="'$(CoverageReportDir)' == ''">$(TestWorkingDir)coverage\</CoverageReportDir>
 
-    <RestoreIlPdbsWithHardlinks Condition="'$(RestoreIlPdbsWithHardlinks)' == ''">true</RestoreIlPdbsWithHardlinks>
+    <UseCoverageDedicatedRuntime Condition="'$(UseCoverageDedicatedRuntime)' == ''">true</UseCoverageDedicatedRuntime>
```

Comment:
>UseCoverageDedicatedRuntime [](start = 5, length = 27)

can this be just the Microsoft.NETCore.App version instead of true/false value? and if the version set, we use it, otherwise we'll skip it.

Reply:
I think this will be confusing: it is not clear what is the side effect of each version, having as a bool make clear of the behavior here.

Reply:
I don't have a strong feeling here but I can imagine anyone choose specific version will guarantee it is a good version to work with.